x/participation/types/params_test.go

```diff
@@ -205,6 +210,11 @@ func TestValidateNextTierBenefits(t *testing.T) {
 		tierBenefits TierBenefits
 		err          error
 	}{
+		{
+			name:         "bonded amount lower or equal than compared benefits",
```

Comment:
I make think the test case name should be renamed for this one and the other one below. We mention "bonded" while `MaxBidAmount` values are asserted

Reply:
thanks for spotting this, was a copy-paste mistake